variables.tf

```diff
@@ -169,3 +180,32 @@ variable "storage_account" {
   type        = any
   default     = {}
 }
+
+variable "private_endpoint" {
+  type        = string
+  description = "NAme of private endpoint"
+  default     = ""
+}
+
+variable "private_service_connection" {
+  type        = string
+  description = "Name for private service connection"
+  default     = "test"
+}
+
+variable "private_dns_zone_name" {
+  type        = string
+  description = "Name for private dns zone"
+  default     = "privatelink.azurewebsites.azure.net"
+}
+
+variable "dns_link" {
+  type        = string
+  description = "Name of DNS link connecting private DNS zone to VNet"
+  default     = ""
+}
+
+variable "dns_resource_group_name" {
+  type = string
+  description = "Name of private DNS zone resource group"
+}
```

Comment:
fix pre-commit

Reply:
Added the line below to terratest:

  dns_resource_group_name      = var.dns_resource_group_name